eng/scripts/resolve-package-info.ps1

```diff
@@ -0,0 +1,44 @@
+param (
+    [string] $ParameterTargetingStringSetting,
+    [string] $PackagePropertiesFolder,
+    [string] $IncludeIndirect = "true"
+)
+
+$setting = $ParameterTargetingStringSetting
+$targetingString = $ParameterTargetingStringSetting
+
+Write-Host "I see the following for env:BuildTargetingString"
+Write-Host $env:BuildTargetingString
+Write-Host "I see the following for env:BUILDTARGETINGSTRING"
+Write-Host $env:BUILDTARGETINGSTRING
+
+# users of internal service builds will set variable "BuildTargetingString" at runtime
+# if they want to artificially filter the discovered packages for the build.
+# when azure devops sets the variable, it will make it all capital letters, so we should check that
+if ($env:BUILDTARGETINGSTRING) {
+    $targetingString = $env:BUILDTARGETINGSTRING
+}
+
+if (Test-Path "$PackagePropertiesFolder") {
+    if ($IncludeIndirect -eq "true") {
```

Comment:
I need to update this code to clean it up from a re-use standpoint, but this _works_ the way we need it to now.